src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Image/Upload.html

```diff
@@ -1,3 +1,8 @@
+<div class="resource-navigation">
+    <div class="resource-navigation-breadcrumbs">
+        <a data-ng-href="{{cancelUrl}}">{{"TR__CANCEL" | translate}}</a>
+    </div>
+</div>
```

Comment:
This directive is also used on the user profile and was not meant to be used as a page on its own (the solution for euth is a hack really).